src/app/productConfig/productConfigForm/productConfigForm.component.js

```diff
@@ -261,6 +292,10 @@ class ProductConfigFormController {
     this.updateQueryParam({ key: giveGiftParams.month, value: month })
   }
 
+  updateDecimalPlaces () {
+    this.customAmount = this.$filter('number')(this.customAmount, 2)
```

Comment:
Is this a string and not a number? How is the decimal place getting off? Maybe we should parse it as a number if we're getting it from the API as a string... Unless the math is causing rounding errors. You seem to have had to call this a bunch. Maybe this filter belongs in the template instead.

Reply:
It is a number typically, and as a number, it drops the ending `0` (numbers prefer `51.2` over `51.20`). When I put it in the template, it caused real problems with the user typing in the number. It would try to format it on every key stroke which was very annoying.